src/graphql/queries.js

```diff
@@ -496,6 +495,32 @@ module.exports = {
     return winnings;
   },
 
+  leaderboardStats: async (root, { filter, orderBy, limit, skip }, { db: { Votes, Topics } }) => {
+      const result = await Votes.find({});
+      let participantsCount = 0;
+      let totalQTUM = new BigNumber(0);
+      let totalBOT = new BigNumber(0);
+      result.reduce((acc, cur) => {
+        const curAmount = new BigNumber(cur.amount);
+        if (!acc.hasOwnProperty(cur.voterAddress)) {
+          acc[cur.voterAddress] = '';
+          participantsCount++;
+        }
+        if (cur.token === TOKEN.BOT) {
+          totalBOT = new BigNumber(totalBOT).plus(curAmount).toString(10);
```

Comment:
keep it as a BigNumber type the whole time so you dont have to convert toString on every iteration. then on the return you can `totalQTUM.toString(10)` and `totalBOT.toString(10)`.